clang/include/clang/Driver/Options.td

```diff
@@ -2649,6 +2649,11 @@ def fsanitize_undefined_strip_path_components_EQ : Joined<["-"], "fsanitize-unde
   HelpText<"Strip (or keep only, if negative) a given number of path components "
            "when emitting check metadata.">,
   MarshallingInfoInt<CodeGenOpts<"EmitCheckPathComponentsToStrip">, "0", "int">;
+def fno_sanitize_top_hot_EQ
+    : CommaJoined<["-"], "fno-sanitize-top-hot=">,
+      Group<f_clang_Group>,
+      HelpText<"Exclude sanitization for the top hottest fraction of code "
```

Comment:
`fraction of code` is not a fraction of the code. it's a fraction of time (more precisely PGO counters) 

Please feel free to reword, but meaning should be like:
`Exclude sanitization for the top hottest code responsible for the given fraction of PGO counters.`

Reply:
Done